language/examples/document-summarization/summarization_with_docai.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>uncapitalize: Full processor --&gt; full processor</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>That is a direct link and page title for the documentation page (which has a capital letter in the title).</p>